### ISSUE TITLE
psnr has more than 80 on the validation set, is this normal

### ISSUE THREAD

**Issue by chenjiachengzzz** [swz30/MIRNetv2#1]
![image](https://user-images.githubusercontent.com/54382260/165757582-41ec2746-4021-4032-b3dc-9e8235b3ee99.png)

**Comment by Lz-2019317** [comment]
你好，你训练代码跑起来了吗？
报错如下：
`ModuleNotFoundError:` No module named 'basicsr'
这个basicsr是项目文件夹，导入为沙会报错阿？
报错如下：
`ERROR:torch.distributed.elastic.multiprocessing.api:failed (exitcode: 1) local_rank: 0 (pid: 28610) of binary: /home/xtzg/anaconda3/envs/pytorch1.1`
我的设置：
yaml配置文件中也修改shuffle为false
`    # data loader
    use_shuffle: false     # true
    num_worker_per_gpu: 0  # 8
    batch_size_per_gpu: 1  # 8`
train.sh中设置为：
`python -m torch.distributed.run --nproc_per_node=1 --master_port=4321 basicsr/train.py -opt $CONFIG --launcher pytorch`
